.gitignore

```diff
@@ -18,3 +19,9 @@ testfm.egg-info/
 #Packaging
 build/
 dist/
+
+# Dynaconf local files
+/.secrets.yaml
+/.secrets_*.yaml
```

Comment:
Remove since secrets should not be required here for now .

Reply:
Removed secrets 👍